Issue: Windows commands not accessible inside distribution terminal (#940)
MicrosoftDocs/WSL: WSL/troubleshooting.md

options = metadata,uid=1000,gid=1000,umask=0022
```

Please note that adding this command will include metadata and modify the file permissions on the Windows files seen from WSL. Please see the [File System Permissions](./file-permissions.md) for more information.

### Running Windows commands fails inside a distribution

> Some distributions [available in Microsoft Store](install-win10.md#step-6---install-your-linux-distribution-of-choice) are yet not fully compatible to run Windows commands in [Terminal](https://en.wikipedia.org/wiki/Linux_console) out of the box. If you get an error `-bash: powershell.exe: command not found` running `powershell.exe /c start .` or any other Windows command, you can resolve it following these steps:

Reviewer: ```suggestion
If you get an error when running Windows commands such as: `-bash: powershell.exe: command not found` running `powershell.exe /c start .` or any other Windows command, you can resolve it following these steps:
```

> 1. In wsl distribution Terminal run `echo $PATH`.  

Reviewer: ```suggestion
1. In your WSL distribution run `echo $PATH`.  
```

>    If it does not feature `/mnt/c/Windows/system32` something is redefining the standard PATH variable.

Reviewer: ```suggestion
   If it does not include: `/mnt/c/Windows/system32` something is redefining the standard PATH variable.
```

2. Check profile settings with `cat /etc/profile`.  

>    If it contains assignment of PATH variable, run this command `sudo sed -i 's/PATH/NULL/g' /etc/profile` or edit manually the file with one of Linux Terminal text editors, ie `sudo nano /etc/profile`, commenting out PATH block with **#** character.

Reviewer: ```suggestion
   If it contains assignment of the PATH variable, edit the file to comment out PATH assignment block with a **#** character.
```
